blockindex/contractstaking/delta_state_test.go

```diff
@@ -0,0 +1,46 @@
+package contractstaking
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/require"
+)
+
+func TestDeltaState_Transfer(t *testing.T) {
+	require := require.New(t)
+
+	cases := []struct {
+		name     string
+		state    deltaState
+		action   deltaAction
+		expected deltaState
+		isError  bool
```

Comment:
change to `errString string` which is specific error for this test
`invalid delta state 1`, `invalid delta action 2 on state 1`, etc

Reply:
fixed